pkg/controller/workspacerouting/solvers/openshift_oauth_solver.go

```diff
@@ -15,7 +15,7 @@ package solvers
 import (
 	"fmt"
 
-	"github.com/che-incubator/che-workspace-operator/pkg/apis/workspace/v1alpha1"
+	"github.com/che-incubator/che-workspace-operator/pkg/apis/controller/v1alpha1"
```

Comment:
Wow. Seems we need to rename module and replace all imports with new operator location )